client/src/scenes/TreePermits.jsx

```diff
@@ -124,15 +124,14 @@ class TreePermits extends Component {
 							this
 						)}
 					/>
-					<Typography variant="paragraphText" mobileVariant="paragraphText"
```

Comment:
this changes the typography font.
can't say that i understand why it worked until now, but along with deleting the variants font-family should be set to Assistant

Reply:
Hmm.... I don't think this code change has any effect on rendering.
There are two `Typography` components in this project - `"@material-ui/core/Typography"` and a custom one in meireim's code), is there a mixup here?